yt/utilities/answer_testing/framework.py

```diff
@@ -63,6 +81,332 @@
 _latest_local = ytcfg.get("yt", "local_standard_filename")
 _url_path = ytcfg.get("yt", "answer_tests_url")
 
+
+# New AnswerTestclass
+class AnswerTest():
```

Comment:
That makes sense. I've done as you suggested. I only had them together in a class because I had originally been applying a class-level fixture to the AnswerTest class, but that is no longer the case so there's no longer a need for the AnswerTest class. The changes will show up in the next push.